docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -3,42 +3,88 @@ title: Set Up Code Builder
 lang: en
 ---
 
-## Install the Code Builder Managed Package
+Enable Code Builder to provide the permissions needed to install the Code Builder managed package in a Salesforce supported org.
 
-Salesforce Code Builder is a second-generation (2GP) managed package that a user with requisite permissions can install in any supported Salesforce org edition.
+### Required Editions
 
-1.  Click the install [link](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000HddeKUAR) to access the Code Builder managed package on AppExchange.
-2. Click **Get It Now**.
-3. Select the org in which you want to install Code Builder.
-4. Check terms and conditions and click **Confirm and Install**.
+**Available in**: Lightning Experience in Enterprise, Performance, Professional and Unlimited Editions.
 
+**Available in**: Government Cloud Plus as interoperable. Connecting Code Builder to Government Cloud Plus orgs can send data outside the authorization boundary. Contact your Salesforce account executive for more details.
 
-![Install Button](../../../images/install_button.png)
+**Inoperable in**: Government Cloud. Although Code Builder appears in the Setup menu, don’t enable or install it because it won’t operate properly.
 
-Grant access and click **Continue**.
+**Not available in**: EU Operating Zone. EU Operating zone is a special paid offering that provides an enhanced level of data residency commitment. Code Builder is supported in orgs in the EU that aren’t part of EU OZ, per standard product terms and conditions.
```

Comment:
Unrelated, but @sbudhirajadoc let's add this piece in for the Einstein for Devs product as well :) We have the same restrictions in place currently.

Reply:
done